apollo-backend/src/main/java/io/logz/apollo/dao/DeploymentDao.java

```diff
@@ -15,6 +15,8 @@
     List<Deployment> getAllRunningDeployments();
     List<Deployment> getAllOngoingDeployments();
     List<Deployment> getAllStartedDeployments();
+    List<Deployment> getAllOngoingDeploymentsByServiceIds(List<Integer> serviceIds);
```

Comment:
this function is not in use

Reply:
@matvey-mtn  True, it is just dao function so I do not think it is critical.